lib/Sema/TypeCheckDecl.cpp

```diff
@@ -2285,11 +2286,13 @@ static Optional<ObjCReason> shouldMarkAsObjC(TypeChecker &TC,
   // explicitly declared @objc.
   if (VD->getAttrs().hasAttribute<ObjCAttr>())
     return ObjCReason::ExplicitlyObjC;
-  // @IBOutlet, @IBAction, and @NSManaged imply @objc.
+  // @IBOutlet, @IBAction, @IBInspectable, @NSManaged imply @objc.
   if (VD->getAttrs().hasAttribute<IBOutletAttr>())
     return ObjCReason::ExplicitlyIBOutlet;
   if (VD->getAttrs().hasAttribute<IBActionAttr>())
     return ObjCReason::ExplicitlyIBAction;
+  if (VD->getAttrs().hasAttribute<IBInspectableAttr>())
```

Comment:
Huh, how did we miss this one? I guess cause there's no *inherent* reason for IBInspectable to imply "exposed to ObjC".